docs-src/concepts/what-is-a-start-to-close-timeout.md

```diff
@@ -20,9 +20,18 @@ A Start-To-Close Timeout is the maximum time allowed for a single [Activity Task
 **The default Start-To-Close Timeout is the same as the default [Schedule-To-Close Timeout](/concepts/what-is-a-schedule-to-close-timeout).**
 
 An Activity Execution must have either this timeout (Start-To-Close) or the [Schedule-To-Close Timeout](/concepts/what-is-a-schedule-to-close-timeout) set.
-We recommend always setting this timeout; however, make sure that it is always set to be longer than the maximum possible time for the Activity Execution to take place.
+We recommend always setting this timeout; however, make sure that Start-To-Close Timeout is always set to be longer than the maximum possible time for the Activity Execution to complete.
 For long running Activity Executions, we recommend also using [Activity Heartbeats](/concepts/what-is-an-activity-heartbeat) and [Heartbeat Timeouts](/concepts/what-is-a-heartbeat-timeout).
 
+:::caution
```

Comment:
Caution is so scary...

Can easily make this a happy `info` or `success` admonition.
We want it to be a good thing :)